<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>regWindow</class>
 <widget class="QWidget" name="regWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>regWindow</string>
  </property>
  <widget class="QLabel" name="introText">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>511</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
     <pointsize>27</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
    border: none;
    background: none;
    color: rgb(100, 249, 189);
    font-family: 'Yu Gothic UI Light', sans-serif;
}</string>
   </property>
   <property name="text">
    <string>Введите свой логин и пароль</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel {
    border: none;
    background: none;
    text-shadow: 0 0 5px rgba(0, 255, 0, 0); /* Initial transparent green glow */
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../resources/bg1.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="neon">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>20</y>
     <width>341</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel {
    border: none;
    background: none;
    text-shadow: 0 0 5px rgba(0, 255, 0, 0); /* Initial transparent green glow */
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../resources/neon.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgba(83, 208, 158, 0.2); 
    border: 2px solid #53D09E;
    border-radius: 20px;
	font-family: 'Yu Gothic UI Light', sans-serif;
    font-size: 20px;
text-align: center;
}
QPushButton:hover {
    color: rgba(83, 208, 158, 1)
}</string>
   </property>
   <property name="text">
    <string>Подтвердить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>430</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgba(83, 208, 158, 0.2); 
    border: 2px solid #53D09E;
    border-radius: 20px;
	font-family: 'Yu Gothic UI Light', sans-serif;
    font-size: 20px;
text-align: center;
}
QPushButton:hover {
    color: rgba(83, 208, 158, 1)
}</string>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLabel" name="introText_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>371</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
     <pointsize>40</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
    border: none;
    background: none;
    color: rgb(100, 249, 189);
    font-family: 'Yu Gothic UI Light', sans-serif;
}</string>
   </property>
   <property name="text">
    <string>Регистрация</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginField">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: rgba(83, 208, 158, 0.2); 
    border: 2px solid #53D09E;
    border-radius: 20px;
    padding: 5px; /* Добавление отступа для текста */
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordField">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: rgba(83, 208, 158, 0.2); 
    border: 2px solid #53D09E;
    border-radius: 20px;
    padding: 5px; /* Добавление отступа для текста */
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="login">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>253</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel {
    border: none;
    background: none;
    text-shadow: 0 0 5px rgba(0, 255, 0, 0); /* Initial transparent green glow */
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../resources/user-interfacew.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="password">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>346</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel {
    border: none;
    background: none;
    text-shadow: 0 0 5px rgba(0, 255, 0, 0); /* Initial transparent green glow */
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../resources/padlockw.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>neon</zorder>
  <zorder>applyButton</zorder>
  <zorder>cancelButton</zorder>
  <zorder>introText_2</zorder>
  <zorder>loginField</zorder>
  <zorder>introText</zorder>
  <zorder>passwordField</zorder>
  <zorder>login</zorder>
  <zorder>password</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
